### DISCUSSION TITLE
Build Darwinia subDAOs

### DISCUSSION THREAD

**Discussion by boundless-forest** [darwinia-network/darwinia#1397] [Ideas]
# Motivation

The suggestion to rename the current Crab chain to Crab SubDAO on https://github.com/orgs/darwinia-network/discussions/1386 is seen as a significant step forward in clarifying the relationship between the Darwinia chain and the Crab chain for ecosystem members. As the canary network for the Darwinia chain, Crab aims to provide a testing ground for innovative technology and unstable features, making it a crucial component of the Darwinia network system. With more ecosystem entities like Crab chain becoming involved, it's important to establish a strong and clear connection between the Darwinia core chain and these entities in order to:
- Simplify understanding for ecosystem members, particularly newcomers.
- Maintain a clear overall system design to avoid unnecessary complexity.
- Ensure that the development goals of the entities align with the core values of the Darwinia core.
- Establish strong connections between the Darwinia core and the ecosystem entities, leading to improved resource utilization and mutual benefits, and promoting the overall system development in a positive direction.
Taking inspiration from the [Maker SubDAO](https://endgame.makerdao.com/endgame/overview#subdaos) design, a proposal is made to work towards realizing this vision.

# Proposal

In our vision, the Darwinia network comprises a core and multiple decentralized entities developed by ecosystem members. The core is dedicated to building the foundational components of the entire system, including a cross-chain general protocol that offers stable cross-chain support to ecosystem projects, as well as a governance framework that facilitates mutual benefits between the core and these entities during the development process. Other entities will have varying degrees of connection to the core, developing their own features and providing services to users. These entities are expected to be governed by a decentralized DAO, referred to as the SubDAO of the Darwinia core, such as Crab SubDAO, DC SubDAO, and so on. The relationship between the core and subDAOs is akin to that of MakerDAO and its subDAOs.

## Governance System

Given that the Darwinia Core serves as the foundational layer of the entire system, its changes can impact all SubDAOs built on top of it. Therefore, we encourage all community members to participate in Core governance by locking their RING tokens and voting to determine the next direction. In terms of subDAO governance, each subDAO has the flexibility to design its own governance system, using either its subDAO token, the core KTON token, or a hybrid approach. I recommend prioritizing the use of KTON as the governance token for the subDAO. KTON tokens are obtained by locking RING tokens in the core for a specific duration. By using KTON as the governance token, all members in the core system can participate in the SubDAO's governance. This approach not only enhances the decentralization of SubDAO governance, but also ensures that the subDAO's development goals align with the core values, enabling mutual benefits between the core and subDAO during their development.

## Emission

As the kernel of the entire system, the Darwinia core will allocate RING tokens to these subDAOs. This allocation mechanism already exists in the current system, where projects can request fund support from the Darwinia treasury. One difference in this proposal compared to previous approaches is the method for determining the amount allocated by the core. Under this proposal, and based on the Governance System design, the core may issue RING tokens to distribute among these subDAOs, based on the amount of locked KTON in their governance system. The more KTON locked in the subDAO, the greater the amount of RING tokens they can receive.

**Comment by SasoLithops** [comment]
Very nicely presented @boundless-forest. This should be also a medium article to help community understand difference between Darwinia Core chain and SubDAO entities.

I would definitely support DAOs autonomous decision to choose a token for their governance but I also agree with advice to except KTON as governance token. This would show commitment to Darwinia as we all know KTON is not easy to obtain and it is a commitment token. 

However first we must allow locked Ring to participate in Darwinia Core chain governance to really make this push as currently only free Ring tokens can vote. I know now this change  can only happen once we implement OPENGOV.

Giving more usecases to Kton holders has been a dream for many of us community members. With these foundations set, hopefully we community also get more involved and not just put all the pressure on Darwinia Dev team.

**Reply by boundless-forest** [reply]
> However first we must allow locked Ring to participate in Darwinia Core chain governance to really make this push as currently only free Ring tokens can vote. I know now this change can only happen once we implement OPENGOV.

As far as I know, the Darwinia OpenGov is on the way and has been rolled out in the Crab chain.

**Reply by SasoLithops** [reply]
Yes you are right it has. Darwinia chain should be soon next I think. And then we should begin the new governance model jurney.

**Comment by hackfisher** [comment]
>  I recommend prioritizing the use of KTON as the governance token for the subDAO. KTON tokens are obtained by locking RING tokens in the core for a specific duration. By using KTON as the governance token, all members in the core system can participate in the SubDAO's governance.

I disagree, for Crab SubDAO, we already have CRAB token, CRAB should be used as the gov token of Crab SubDAO. Each SubDAO should farming their own token and building SubElixir relationships with RING.

Again, every toke economic‘s motivation is following the token itself at its core, not the project, if a DAO is governance by another DAO's gov token, Governance goals will be confused and be fucked up.

**Reply by SasoLithops** [reply]
I read about SubElixir in MkrDAO. 
So to understand each other KTON would only be used for getting certain ammount of RING from Darwinia Core and you think it is best each DAO uses own token for governance.

I see your point in KTON not being used in governance for each subDAO now as this really could get messed up. Each DAO I have participated until now actually had only one token so I need to learn.I was not familiar with MkrDAO.

But maybe you could add KTON in Darwinia OPENGOV next to Ring. As othervise the ones with holding and staking mostly Kton fall out of governance. Is that actually executable?
